deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -1364,6 +1364,11 @@ def reset_deploy(request, name, stage, host_id):
     agents_helper.retry_deploy(request, name, stage, host_id)
     return HttpResponse(json.dumps({'html': ''}), content_type="application/json")
 
+# retry all deploys for this host
+def reset_all_deploys(request, name, stage, host_id):
```

Comment:
This is a good suggestion, but as far as our users will think about "resetting environments", because their services were not deployed and not about resetting agent, because users maybe don't need to think about internals of our system, I'm proposing to rename it to `reset_all_environments` instead of `reset_all_agents`.

Also host will have only one agent. But my point here is not strict, so I can rename `reset_all_environments` to `reset_agent`.